.github/workflows/ci.yml

```diff
@@ -273,9 +273,9 @@ jobs:
         echo ${GITHUB_SHA} | tee onefuzzlib/git.version
         python -m pip install --upgrade pip
         pip install --target="./.python_packages/lib/site-packages" -r ./requirements.txt ${GITHUB_WORKSPACE}/artifacts/sdk/onefuzztypes-*.whl
-        zip -r api-service.zip .
+        zip -r api-service-py.zip .
```

Comment:
I think we can eliminate this now, right?

Reply:
Yup I left on purpose to see if anyone had thoughts :) I'm also on the side of removing.